content/en/docs/instrumentation/go/sampling.md

```diff
@@ -1,36 +1,45 @@
 ---
 title: Sampling
 weight: 80
-cSpell:ignore: sdktrace
 ---
 
-Sampling is a process that restricts the amount of traces that are generated by
-a system. The exact sampler you should use depends on your specific needs, but
-in general you should make a decision at the start of a trace, and allow the
+Sampling is a process that restricts the amount of spans that are generated by a
```

Comment:
Recommend linking to the concept page, `/docs/concepts/sampling/`

Reply:
https://github.com/open-telemetry/opentelemetry.io/pull/3318/commits/dc2a8087756660335c32a696436bbe6836a466b5